lib/secure_headers/railtie.rb

```diff
@@ -3,9 +3,22 @@
   module SecureHeaders
     class Railtie < Rails::Engine
       isolate_namespace ::SecureHeaders if defined? isolate_namespace # rails 3.0
+      conflicting_headers = ['X-Frame-Options', 'X-XSS-Protection', 'X-Content-Type-Options',
+                             'X-Permitted-Cross-Domain-Policies', 'X-Download-Options',
+                             'X-Content-Type-Options', 'Strict-Transport-Security',
+                             'Content-Security-Policy', 'Content-Security-Policy-Report-Only',
+                             'X-Permitted-Cross-Domain-Policies']
```

Comment:
Good point on that @oreoshake RE: adding the other headers to future proof it. Might also be nice if there was a way to reach in and grab the header names directly from each `Header` class, but I couldn't think of a good way to do it.